embassy-nrf/src/pwm.rs

```diff
@@ -216,15 +207,43 @@ impl<'d, T: Instance> SequencePwm<'d, T> {
         r.tasks_stop.write(|w| unsafe { w.bits(0x01) });
     }
 
+    /// Enables the PWM generator.
+    #[inline(always)]
+    pub fn enable(&self) {
+        let r = T::regs();
+        r.enable.write(|w| w.enable().enabled());
+    }
+
     /// Disables the PWM generator.
     #[inline(always)]
     pub fn disable(&self) {
         let r = T::regs();
         r.enable.write(|w| w.enable().disabled());
     }
+
+    /// Sets duty cycle (15 bit) for a PWM channel.
+    pub fn set_duty(&mut self, channel: usize, duty: u16) {
+        let r = T::regs();
+
+        self.sequence[channel] = duty & 0x7FFF;
+
+        r.seq0
+            .ptr
+            .write(|w| unsafe { w.bits(self.sequence.as_ptr() as u32) });
+
+        // defensive before seqstart
+        compiler_fence(Ordering::SeqCst);
+
+        // tasks_seqstart() doesn't exist in all svds so write its bit instead
+        r.tasks_seqstart[0].write(|w| unsafe { w.bits(1) });
+
+        // defensive wait until waveform is loaded after seqstart
+        while r.events_seqend[0].read().bits() == 0 {}
+        r.events_seqend[0].write(|w| w);
+    }
```

Comment:
Dropped the `max_duty`/`set_max_duty`, `period`/`set_period`, and `prescaler`/`set_prescaler` methods as we now have `Config`.

Reply:
sequencepwm cant have set_duty :)
The whole point of sequence pwm is you dont get to control duty after its started (unless you stop and restart it)